src/dll.rs

```diff
@@ -113,7 +113,7 @@ pub extern "stdcall" fn DllMain(
 pub unsafe extern "system" fn DllGetClassObject(
     rclsid: *const GUID,
     riid: *const GUID,
-    pout: *mut *const core::ffi::c_void,
+    pout: *mut *mut core::ffi::c_void,
```

Comment:
Oh, unknown.query now accepts mut mut. Seems it also accepts `*const GUID` instead of a reference now, can you remove `&*` there while we are at it?

Reply:
Good catch!